.github/workflows/lint.yaml

```diff
@@ -2,17 +2,22 @@
 name: Linting and Formatting
 
 on:  # yamllint disable-line rule:truthy
-  pull_request:
-
+  pull_request: {}
+  workflow_dispatch: {}
 jobs:
   shell:
-    uses: ASFOpenSARlab/osl-utils/.github/workflows/reusable-shell-code-quality.yaml@v0.0.3
-
+    uses: ASFOpenSARlab/osl-utils/.github/workflows/reusable-code-quality-shell.yaml@v0.0.9
+    with:
+      osl-utils-tag: v0.0.9
```

Comment:
This is something that I've been mulling over for a bit and couldn't quite see a path forward... `osl-utils-tag` is confusing to me in this context because without looking at the docs (and even _after_ reading the README in osl-utils, its not obvious if this is a github release tag, or a image container tag. 

I KNOW it'd require some retooling, but could we move towards using `osl-utils-release` or `osl-utils-image-tag` to disambiguate?